src/System.ComponentModel.Composition/tests/System/ComponentModel/Composition/CompositionErrorIdTests.cs

```diff
@@ -10,7 +10,6 @@ namespace System.ComponentModel.Composition
     public class CompositionErrorIdTests
     {
         [Fact]
-        [ActiveIssue(25498, TargetFrameworkMonikers.UapAot)]
```

Comment:
Should we also close issues related with these attributes? i.e: https://github.com/dotnet/corefx/issues/25498

Reply:
I think so. If it will be a lot of time possibly vendors coudl help.

Reply:
That was the idea. I've already spent too much time on this PR as I went the extra mile and also removed tons of dead code. I would like hand this over to vendors is possible. I think overall it makes sense to go over open test issues in corefx. Probably half of them can be closed already.

Reply:
I closed like 8-10 issues as I was reviewing the PR (Mostly the desktop once) so yes, if possible some help from vendors would be great.